### ISSUE TITLE
Negative result for measure

### ISSUE THREAD

**Issue by kenn** [zverok/time_math2#6]
It seems that `TimeMath.measure` does not handle negative calculations well.

The following example should return `-1 day`. What do you think?

```ruby
> a = Date.parse('2017-06-27')
> b = Date.parse('2017-06-26')

> TimeMath.measure(b,a)
=> {:years=>0,
 :months=>0,
 :weeks=>0,
 :days=>1,
 :hours=>0,
 :minutes=>0,
 :seconds=>0}

> TimeMath.measure(a,b)
=> {:years=>-1,
 :months=>11,
 :weeks=>4,
 :days=>2,
 :hours=>0,
 :minutes=>0,
 :seconds=>0}
```

**Comment by kenn** [comment]
Also, zero diff between dates returns a weird value.

```ruby
a = Date.parse('2017-06-27')

> TimeMath.measure(a,a)
=> {:years=>-1, :months=>12, :weeks=>0, :days=>0, :hours=>0, :minutes=>0, :seconds=>0}
```

**Comment by zverok** [comment]
Confirmed. On it.

**Comment by zverok** [comment]
Please check `develop` branch with your use-cases. Currently:
```ruby
a = Date.parse('2017-06-27')
b = Date.parse('2017-06-26')
TimeMath.measure(b,a)
# => {:years=>0, :months=>0, :weeks=>0, :days=>1, :hours=>0, :minutes=>0, :seconds=>0}
TimeMath.measure(a,a)
# => {:years=>0, :months=>0, :weeks=>0, :days=>0, :hours=>0, :minutes=>0, :seconds=>0}
TimeMath.measure(a,b)
# => {:years=>0, :months=>0, :weeks=>0, :days=>-1, :hours=>0, :minutes=>0, :seconds=>0}
```
OK?

**Comment by kenn** [comment]
Perfect, thank you! 👍

**Comment by zverok** [comment]
Thanks for the report. Released 0.0.8 version with this (and some other small) fixes.
